gee/landpks/src/main.py

```diff
@@ -366,7 +366,7 @@ def base_image(year, geojson, lang, gc_client, metadata):
 
     # Reproject L8 image so it can retrieve data from every latitute 
     # without deforming the aoi bounding box
-    l8sr_mosaic_bands = l8sr_y.reproject(scale=30, crs='EPSG:3857')
+    map_l8sr_mosaic = map_l8sr_mosaic.reproject(**({'crs':'EPSG:3857','scale':30}))
```

Comment:
There is no need for the ** notation here to set the parameters from a dictionary (it is kind of a holdover from Javascript). In Python this line can just be:

`map_l8sr_mosaic = map_l8sr_mosaic.reproject(crs='EPSG:3857', scale=30)`